trivy.yaml

```diff
@@ -0,0 +1,2 @@
+vulnerability:
```

Comment:
ooh! did not know this could be done

Reply:
FYI this is not a real vulnerability, it's Code Factor being paranoid.
We have `grpcio` pinned to a safe version, but ray has it pinned to `>=x`, where `x` is an unsafe version.
I can do nothing to change that, apart from adding this `trivy` file